src/components/progressCircular/progressCircular.scss

```diff
@@ -82,6 +81,8 @@ md-progress-circular {
     }
   }
 
+  // TODO: This while-loop generates about 2 kilobytes of css after gzip.
+  // Refactor progressCircular to animate with javascript.
```

Comment:
I created #1926. I'm going to consult w/ Matias next week to see if he has an idea for a simple way to do this (I tried using $animate.animate directly and it was somewhat choppy).